aiida_pseudo/groups/family/pseudo_dojo.py

```diff
@@ -3,6 +3,7 @@
 import collections
 import json
 import os
+import pathlib
```

Comment:
In `aiida_pseudo/cli/utils.py` you only importh `Path`:

```Python
from pathlib import Path
```

Since no other methods/classes from `pathlib` are used in this module, perhaps do the same here to appease my OCD? 😁

Reply:
but then my OCD will be triggered:
```
import collections
import json
import os
from pathlib import Path
```
is so fugly...

Reply:
Then move it down a few lines next to

```Python
from typing import Sequence
```

? 😁

Reply:
Your wish is my command